docs/data-sources.md

```diff
@@ -25,8 +25,8 @@ The underlying data sources are a mix of:
 
 These source images are composited to form tiles that make up the Mapzen Terrain Tiles service. To determine exactly which images contributed to Mapzen Terrain Tiles in a particular area, you can download the footprints database and use it with a GIS program like [QGIS](http://www.qgis.org/) to inspect which of these sources were used.
```

Comment:
Let's insert an image preview here by adding the following image to the https://github.com/tilezen/joerd/tree/master/docs/images directory and referencing it here with relative path:

<img width="1004" alt="screen shot 2017-08-01 at 13 34 38" src="https://user-images.githubusercontent.com/853051/28845866-4b8daa90-76be-11e7-8f9d-10ecd475bc04.png">

Here's the QGIS layer file for future fiddling (store in https://github.com/tilezen/joerd/tree/master/docs/examples):
[footprints-terrain-v2.qml.zip](https://github.com/tilezen/joerd/files/1191855/footprints-terrain-v2.qml.zip)